about/structure.md

```diff
@@ -58,3 +60,44 @@ You can find a list of teams below.
 ## International Computer Science Institute (ICSI)
 
 [ICSI](https://icsi.berkeley.edu) is the host organization of 2i2c. It provides administrative and strategic resources for 2i2c and leadership to support them in their mission. ICSI supports 2i2c by assisting in managing partnerships with 2i2c collaborators and contracts with 2i2c customers, connecting with the larger open research and education community, fundraising, grant and financial management, assuming legal and financial risk for 2i2c, and hiring and management of staff.
+
+## Scope of governance
+
+There are several kinds of governance scopes within 2i2c, described below.
+
+### Decisions that require a full steering council vote
+
+The [Steering Council](structure:steerco) defines the strategy and major direction of 2i2c. It must vote on major decisions that have strong implications for 2i2c's strategy or financial well-being. See [the Steering Council structure page](structure:steerco) for information about how it votes.
+
+Here are some major decision areas that require a full steering council vote.
+
+- Unplanned financial decisions over $20,000 (e.g., deciding to hire a contractor that has not been written into a grant)
+- Decisions that have significant implication for 2i2c's financial health.
+- Major changes in strategic direction and roadmaps for business or technical development.
```

Comment:
This could cross reference the strategic planning document and related terms (focus areas, projects, deliverables, tasks, etc.) Which level requires SC approval?